ts/features/euCovidCert/screens/valid/EuCovidCertQrCodeFullScreen.tsx

```diff
@@ -0,0 +1,68 @@
+import { View } from "native-base";
+import * as React from "react";
+import {
+  Dimensions,
+  Image,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet
+} from "react-native";
+import { NavigationInjectedProps } from "react-navigation";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import I18n from "../../../../i18n";
+import { useMaxBrightness } from "../../../../utils/brightness";
+import { cancelButtonProps } from "../../../bonus/bonusVacanze/components/buttons/ButtonConfigurations";
+
+type NavigationParams = Readonly<{
+  qrCodeContent: string;
+}>;
+
+const styles = StyleSheet.create({
+  qrCode: {
+    // TODO: it's preferable to use the hook useWindowDimensions, but we need to upgrade react native
+    width: Dimensions.get("window").width,
+    height: Dimensions.get("window").width,
+    flex: 1
+  }
+});
+
+export const EuCovidCertQrCodeFullScreen = (
+  props: NavigationInjectedProps<NavigationParams>
+): React.ReactElement => {
+  useMaxBrightness();
+  return (
+    <BaseScreenComponent goBack={true}>
```

Comment:
```suggestion
    <BaseScreenComponent goBack={true} contextualHelp={emptyContextualHelp}>
```
Could it be useful?

Reply:
I didn't insert it because by design it is not provided for this screen 
<img src=https://user-images.githubusercontent.com/26501317/120976250-70f16700-c772-11eb-8e3a-943ec9b2ed4c.png width=300 />